Services/ArticlesApiService.js

```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
+
+// const url = `http://192.168.1.178:3000`;  // whenever we want to make api calls to localhost we have to use the ip address not the keyword `localhost` since that can result in a network error.
```

Comment:
why do you need to keep this commented out code?

Reply:
Removed